cli/waiter/token_post.py

```diff
@@ -152,6 +164,15 @@ def add_token_flags(parser):
     parser.add_argument('--ports', help='number of ports to reserve for service', type=int)
 
 
+def add_override_flags(parser):
+    """Adds the arguments override file values flags to the given parser"""
+    override_group = parser.add_mutually_exclusive_group(required=False)
+    override_group.add_argument('--override', action='store_true', dest='override',
+                                help='Allow overriding values in file with values from other arguments. '
+                                     'Overriding values is disallowed by default.')
+    override_group.add_argument('--no-override', action='store_false', dest='override', help=argparse.SUPPRESS)
```

Comment:
Is this necessary if the default is to not do an override?

Reply:
Added it for completeness for boolean arguments, I'm okay dropping it if you do not like it.

Reply:
Removed `--no-override` argument.

Reply:
FWIW, Click (a popular alternative CLI library to argparse) recommends keeping both options. This allows a caller who wants to be explicit about their `--no-override` preference to do so today, and their invocation will continue to behave the same way, even if the Waiter CLI changes its default behavior to `--override` in the future.

https://click.palletsprojects.com/en/7.x/options/#boolean-flags
> Click always wants you to provide an enable and disable flag so that you can change the default later.

That said, if there are any usage metrics about how often the "You cannot specify the same parameter in both..." error is occurring, it would be worth consulting those. If the only occurrences in the past N months correspond to our own usage, then I think we could safely flip the default behavior to `--override` today without breaking anyone.

Sorry for the late comment, I only just saw this!

Reply:
And thanks for getting this improvement in so quickly! Shams ftw!

Reply:
Adding both the options back, `--override` and `--no-override`.